pysteps/blending/steps.py

```diff
@@ -1451,10 +1456,52 @@ def worker(j):
                         # forecast outside the radar domain. Therefore, fill these
                         # areas with the "..._mod_only" blended forecasts, consisting
                         # of the NWP and noise components.
+
                         nan_indices = np.isnan(R_f_new)
-                        R_f_new[nan_indices] = R_f_new_mod_only[nan_indices]
-                        nan_indices = np.isnan(R_pm_blended)
-                        R_pm_blended[nan_indices] = R_pm_blended_mod_only[nan_indices]
+                        if smooth_radar_mask_range != 0:
+                            # Compute the smooth dilated mask
+                            new_mask = blending.utils.compute_smooth_dilated_mask(
+                                nan_indices,
+                                max_padding_size_in_px=smooth_radar_mask_range,
+                                gaussian_kernel_size=9,
```

Comment:
So this value doesn't need to be changed by the user?

Reply:
Which value are you talking about here? The kernel size or de range? In the case of the first, it can be changed but should be experimented with according to the domain and radar observation. In the case of nr 2, the method should provide this as a parameter.

Reply:
I was referring to `gaussian_kernel_size` which is hardocoded

Reply:
Just discussed with @sidekock, the `gaussian_kernel_size` ensures that the nan-fields are large enough to start the smoothing. Without it, you would also apply this to local nan-values in the radar domain (if these occur - which is the case for the Belgian data). The fixed value does the trick. It is the default in the `blending.utils.py` functionality, so it does not need to be repeated here (hence, `gaussian_kernel_size = 9` can be removed here in my opinion).

Reply:
OK thanks @RubenImhoff , I agree, let's remove the hardcoded value and rely on the default